### PR TITLE
[compiler] Support operators with multiple outputs

### PR DESCRIPTION
**Pull request by bwasti**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #24 [WIP] Use PackedFunc to enable custom TVM operator registration from python
* #23 [benchmark] Add initial benchmark file
* #22 [compiler] Improve conversion from PyTorch IR to Relay
* #21 [testing] Improve testing to include models, image inputs and autotuned parameters
* #17 [compiler] Add None type handling with sentinel value
* **#14 [compiler] Support operators with multiple outputs**

### CODE REVIEW

File: torch_tvm/compiler.cpp [hunk 1]
```diff
@@ -108,24 +108,49 @@ tvm::relay::Function TVMCompiler::convertToRelay(
...
+      return value_map[sg_output];
+    }
+    tvm::NodePtr<tvm::relay::TupleNode> n = tvm::make_node<tvm::relay::TupleNode>();
+    tvm::Array<tvm::relay::Expr> fields;
+    for (const auto& sg_output : subgraph->outputs()) {
+      AT_ASSERT(value_map.find(sg_output) != value_map.end());
+      fields.push_back(value_map[sg_output]);
+    }
+    n->fields = std::move(fields);
+    return tvm::relay::Tuple(n);
```
Comment (wanchaol) [inline torch_tvm/compiler.cpp]: can we actually make even the single output also fills into a Tuple, so that we could merge the single and multiple output case together? that way we could avoid checking the output size everytime.

File: torch_tvm/compiler.cpp [hunk 2]
```diff
@@ -108,24 +108,42 @@ tvm::relay::Function TVMCompiler::convertToRelay(
         if (use.user->outputs().size() < 1) {
           continue;
         }
-        // TODO handle multiple outputs
-        AT_ASSERT(use.user->outputs().size() == 1);
-        value_map[use.user->output()] = getOperator(use.user, relay_inputs);
-        new_frontier.emplace_back(use.user->output());
+        // if there are 2+ outputs, getOperator returns a tuple
+        if (use.user->outputs().size() == 1) {
```
Comment (wanchaol) [inline torch_tvm/compiler.cpp]: shouldn't this if can be combine together with the else statement if in the batchnorm we eliminate the conditional on output==1?

**Review by wanchaol — APPROVED** [review]
LGTM, one minor suggestion
